sdk/spring/spring-sdk-client-test.yml

```diff
@@ -0,0 +1,140 @@
+parameters:
+  - name: Artifacts
+    type: object
+    default: []
+  - name: TestPipeline
+    type: boolean
+    default: false
+  - name: AdditionalModules
+    type: object
+    default: []
+  - name: SDKType
+    type: string
+    default: client
+  - name: ServiceDirectory
+    type: string
+    default: spring
+  - name: MatrixConfigs
+    type: object
+    default:
+      - Name: Java_ci_test_base
+        Path: eng/pipelines/templates/stages/platform-matrix.json
+        Selection: sparse
+        NonSparseParameters: Agent
+        GenerateVMJobs: true
+  - name: AdditionalMatrixConfigs
+    type: object
+    default: []
+  - name: MatrixFilters
+    type: object
+    default: []
+  - name: MatrixReplace
+    type: object
+    default: []
+  - name: SkipAggregateReports
+    type: boolean
+    default: false
+  - name: PreBuildSteps
+    type: object
+    default: []
+  - name: SpringBootVersion
+    type: string
+    default: 'not-specified' # Set a default that breaks in obvious ways.
+  - name: SpringCloudVersion
+    type: string
+    default: 'not-specified' # Set a default that breaks in obvious ways.
+
+jobs:
+  - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
```

Comment:
How about not using this template? Just do these steps: 
1. update version
2. inset dependency management
3. run test